packages/accordion-react/src/Accordion.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {ReactNode, useState} from 'react';
+import chevron from "../../accordion/chevron.svg";
```

Comment:
Samme, må impoteres fra @fremtind/jkl-accordion

Reply:
Husk da å bruke `lerna add` for å legge til pakken som dependency, ellers vil den ikke funke som den skal